compiler/optimizations/gpuTransforms.cpp

```diff
@@ -661,8 +661,39 @@ GpuKernel::GpuKernel(const GpuizableLoop &gpuLoop, DefExpr* insertionPoint)
   }
 }
 
+static bool isWrapperSymbol(Symbol* sym) {
+  return sym->hasFlag(FLAG_COBEGIN_OR_COFORALL_BLOCK) ||
+         sym->hasFlag(FLAG_COBEGIN_OR_COFORALL) ||
+         sym->hasFlag(FLAG_ON) ||
+         sym->hasFlag(FLAG_ON_BLOCK);
+}
+
+static Symbol* findParentFunction(Symbol* sym) {
```

Comment:
Add a comment indicating this function isn't reliable in the face of things like inlining.